features.md

```diff
@@ -0,0 +1,151 @@
+## Features
+
+### Code Completions
+
+**Solidity by Nomic Foundation** autocompletes references to existing symbols (e.g. contract instances, globally available variables and built-in types like arrays) and import directives (i.e. it autocompletes the path to the imported file).
```

Comment:
This text works for the vscode extension but is out of place in the language server/coc-solidity extension.

I am not sure on this. Maybe we go for something generic `Autocomplete references` or maybe we parameterise the codegen to swap the name in (e.g. `Solidity by Nomic Foundation` or `@ignored/coc-solidity` or `Solidity language server`)